pkg/images/images_test.go

```diff
@@ -246,17 +246,17 @@ func Test_releaseToKeep(t *testing.T) {
 			name: "2.7.0 < min < max < 2.9.0",
```

Comment:
need to be updated to
```suggestion
			name: "2.7.0 < min < max < 2.10.0",
```